lib/fsevents-handler.js

```diff
@@ -3,8 +3,34 @@
 var fs = require('fs');
 var sysPath = require('path');
 var readdirp = require('readdirp');
-var fsevents;
-try { fsevents = require('fsevents'); } catch (error) {}
```

Comment:
just keep it up here and leave it simple

```js
catch (error) {
  if (process.env.CHOKIDAR_PRINT_FSEVENTS_REQUIRE_ERROR) console.log(error)
}
```

Why do anything more than that? The rest is overkill.